### PR TITLE
[IMP] website: introduce `s_image_title` snippet

### PR DESCRIPTION
**Pull request by Brieuc-brd**
This commit adds the new `s_image_title` snippet.

task-4105319
Part of task-4077427

Requires:
- https://github.com/odoo/design-themes/pull/867

Capture d’écran 2024-08-13 à 16 09 58

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by xlu-odoo — APPROVED** [review]
LGTM! Nice work

**Comment by Brieuc-brd** [comment]
Rebased + added custom texts for themes.

**Review by SergeBayet — COMMENTED** [review]
LGTM :+1: . Just a comment about the snippet options. See below.
I'm starting the review of the design-themes PR.

File: addons/website/views/snippets/s_image_title.xml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                <we-button title="Align Middle"
+                        data-select-class="justify-content-center"
+                        data-img="/website/static/src/img/snippets_options/align_middle.svg"/>
+                <we-button title="Align Bottom"
+                        data-select-class="justify-content-end"
+                        data-img="/website/static/src/img/snippets_options/align_bottom.svg"/>
+            </we-button-group>
+        </div>
+    </xpath>
+</template>
```
[the remark this answers is not in the document]
Comment (Brieuc-brd) [reply addons/website/views/snippets/s_image_title.xml]: Good catch, thanks ! I've updated the icons and applied the point 2/c :)
The diff : https://github.com/odoo/odoo/compare/ce7f45add17f2c2342dbc21bd70f5a2042b40d5a..cb51055a66a1cfbf7a60c4be634a785d1962b6b1

**Comment by Brieuc-brd** [comment]
Hey @SergeBayet 👋 
Thanks for your review 👍 I've updated both branches, could you have a look ? 
Thanks in advance !

File: addons/website/views/snippets/s_image_title.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (SergeBayet) [reply addons/website/views/snippets/s_image_title.xml]: @sobo-odoo Seems good to me. Do you agree? If so you can delegate or r+ yourself. Thanks in advance.

**Review by SergeBayet — APPROVED** [review]
LGTM. :1st_place_medal:

File: addons/website/views/snippets/s_image_title.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (sobo-odoo) [reply addons/website/views/snippets/s_image_title.xml]: @SergeBayet If it is okay with you then I don't have to agree with you or double check :wink: I only delegate the PR so once you feel everything is okay, you can r+ :slightly_smiling_face:

**Comment by sobo-odoo** [comment]
@robodoo delegate=SergeBayet

**Comment by SergeBayet** [comment]
@robodoo r-

**Comment by Brieuc-brd** [comment]
@SergeBayet 👋 
Conflicts resolved 🟢

**Comment by SergeBayet** [comment]
@robodoo  r+

**Comment by SergeBayet** [comment]
@robodoo r-

**Comment by SergeBayet** [comment]
@Brieuc-brd See my comment on the design-theme PR. :+1:

**Comment by SergeBayet** [comment]
@robodoo r+

**Pull request closed without merging**